.github/dependabot.yaml

```diff
@@ -785,6 +785,16 @@ updates:
           - "version-update:semver-patch"
       - dependency-name: "example_common"
       - dependency-name: "amplify_lints"
+  - package-ecosystem: "pub"
```

Comment:
is it generated by aft? I think it is missing a section for `packages/logging_cloudwatch/amplify_logging_cloudwatch`. also no GH workflow is added for this package.